pyintegration/deephaven/_wrapper.py

```diff
@@ -70,3 +70,13 @@ def wrap_j_object(j_obj: jpy.JType) -> Union[JObjectWrapper, jpy.JType]:
     wc = _lookup_wrapped_class(j_obj)
 
     return wc(j_obj) if wc else j_obj
+
+
+def unwrap(obj: Any) -> Union[jpy.JType, Any]:
+    """ Returns the wrapped raw Java object if this is a wrapped Java object. Otherwise, returns the same object. """
```

Comment:
Yes, I like this! Were there other places we should use this method instead (I think there might have been a few places that were explicitly checking to unwrap Tables, and maybe some other wrapped types?).

Reply:
Except in deephaven/server/script_session/__init__.py with includes the support of unwrapping legacy's FigureWrapper for compatibility till we retire deephaven_legacy, this is the first case that we would accept a mix of parameter types including the wrapped one which calls for the generic  unwrapping.